internal/domain/rayfile/service.go

```diff
@@ -0,0 +1,38 @@
+package rayfile
+
+import (
+	"io"
+)
+
+type tomlService interface {
```

Comment:
Some information on naming conventions. For such type of interfaces use name `[parentName]TOMLEncoder`, so will be `rayfileTOMLEncoder`

Reply:
I think I got your point. Renamed it do `rayfileTOMLDecoder`

I also have `ioWrapper` there. Should I also change the name or is it okay?

Reply:
yeah, lets keep it with rayfile prefix for now